concurrency-limits-core/src/main/java/com/netflix/concurrency/limits/limiter/AbstractPartitionedLimiter.java

```diff
@@ -105,10 +105,10 @@ public Limiter<ContextT> build() {
 
     static class Partition {
         private final String name;
+        private final AtomicInteger busy = new AtomicInteger(0);
 
         private double percent = 0.0;
         private int limit = 0;
```

Comment:
Calls to `updateLimit` don't appear to guarantee thread-safety. Any reason we shouldn't move this to `AtomicInteger` as well?

Reply:
Can do that, no reason not to.